wiki/Article_Style_Guide/en.md

```diff
@@ -370,104 +424,154 @@ Please limit to a level of four (4) sub-lists at a time (if you need more, you s
-    - `[Events]`
-- the name of a file extension
-  - must add a period (`.`) then the file extension in lowercase letters.
-  - examples:
-    - `.jpg`
-    - `.gif`
-    - `.osz`
-- the name of a chat channel
-  - examples:
-    - `#lobby`
-    - `#osu`
+For the following scenarios, use code.
+
+### Keyboard Keys
+
+When doing this, use capital letters for single characters and [camel case](https://en.wikipedia.org/wiki/Camel_case) for modifers.
+
+For example:
+
+- `` `.` ``
+- `` `Alt` ``
+- `` `Ctrl` + `Shift` + `A` ``
+
+### Button or Menu Text
+
+When copying the text from a menu or button, copy the letter casing as-is.
+For example:
+
+- `` Click `File` then `Save As...` ``
+- `` `1. Let's do it.` ``
+- `` `4. Noooo! I didn't mean to!` ``
+
+### Folder Names
+
+When copying the name of a folder, copy the letter casing as-is.
+For example:
+
+- `` `Exports` ``
+- `` `Downloads` ``
+
+### Directory Names
+
+When copying the path of a directory, copy the letter casing as-is.
+For example:
+
+- `` `Downloads/` ``
+- `` `Skins/User/` ``
+
+### Keywords or Commands
+
+When copying a keyword or command, copy the letter casing as-is or as you would normally type it as (prefer lowercase if applicable).
+For example:
+
+- `` `ComboBurstStyle` ``
+- `` `[Events]` ``
+- `` `git --help` ``
+
+### File Names
+
+When copying the name of a file, copy the letter casing as-is.
+For example:
+
+- `` `osu.exe` ``
+- `` `screenshot001.jpg` ``
+- `` `Opsu-android0.15.0b.apk` ``
+
+### File Extensions
+
+When writing a file extension, prefix the extension with a period (`.`), followed by the file extension in lowercase letters.
+
+- `` `.jpg` ``
+- `` `.exe` ``
+- `` `.osz` ``
+
+### Chat Channels
+
+When copying the name of a chat channel, start it with a hashtag (`#`), followed by the channel name in lower case letters.
+See [Chat Channels](#chat-channels) for other details.
+For example:
+
+- `` `#lobby` ``
+- `` `#osu` ``
 
 ### Code Blocks
 
 When using code blocks, use the ` ``` ` (triple grave mark) syntax.
 
-The osu!wiki site has a feature where you _could_ use four spaces to trigger the code blocks; however, this is **heavily** discouraged.
+Code blocks in the osu!wiki do **not** use syntax highlighting.
+
+Markdown also allows you to use 4 spaces to trigger the code blocks; however, this is **heavily** discouraged.
 
 ## Links
 
+You can use either the reference or inline style links.
+If you are using the reference style linking, it is suggested to place the reference links at the top of the article for quicker access.
+
+When linking to the osu!website (e.g. user profiles or beatmaps), use `osu.ppy.sh` instead of `new.ppy.sh`.
+`new.ppy.sh` is just a temporary subdomain for the new design.
+
+When linking to a osu!user profile, use the user's id number instead of their username.
+Usernames can be changed, while user ids are bound to the user.
+This is to ensure that the link still links to the same person even after they changed their username.
```

Comment:
huh, things have changed since then (or I wasn't keeping up with osu!news) :p...
Maybe add some custom markdown syntax to fetch the username for us instead..?
``[[u/2]]`` -> ``[peppy](https://osu.ppy.sh/u/2)``
🤔 actually that might be bad for when a user is banned :/
---
¯\_(ツ)_/¯, but alrighty 🙏 that they don't keep change their names or whatevs